fiorella/src/sections/header.liquid

```diff
@@ -6,7 +6,95 @@
   {% endif %}
 </style>
 <div data-section-id="{{ section.id }}" data-section-type="header-section">
-  <header role="banner">
+  <header role="banner" class="">
+    <!-- Mobile slide out menu -->
+    <div id="menu" class="menu-container">
+      <div class="menu-top-section">
+        {{ 'logo2_fiorella_mobile.png' | asset_url | img_tag: 'Mobile Logo'}}
+        <div id="hide-menu-btn" class="menu-close">
+          <button type="button" name="btn">{{ 'icon-hamburger.png' | asset_url | img_tag: 'Burguer Menu'}}</button>
+        </div>
+      </div>
+      <div class="search-container">
+        <form action="/search" method="get" role="search">
+          <label for="Search" class="label-hidden">
+            {{ 'general.search.placeholder' | t }}
+          </label>
+          <input type="search"
+                 name="q"
+                 id="Search"
+                 value="{{ search.terms | escape }}"
+                 placeholder="{{ 'general.search.placeholder' | t }}">
+          <button type="submit" class="btn">
+            {% include 'icon-search' %}
+            <span class="icon-fallback-text">{{ 'general.search.submit' | t }}</span>
+          </button>
+        </form>
+      </div>
+      <div class="mobile-nav">
+        <h2>BOUQUETS</h2>
```

Comment:
>You may create collections from Shopify Admin, and later iterating the object `collections` to print `collection.title`. I recommend using an unordered list because collection title should be a link to the collection or a div tag with anchor tags inside

Example:

```
<ul>
  {% for collection in collections %}
    <li><a href="{{ collection.url }}">{{ collection.title }}</a><li>
  {% endfor %}
<ul>
```